sympy/combinatorics/tensor_can.py

```diff
@@ -74,8 +74,8 @@ def dummy_sgs(dummies, sym, n):
 def _min_dummies(dummies, sym, indices):
     """
     Return list of minima of the orbits of indices in group of dummies
-    see `double_coset_can_rep` for the description of `dummies` and `sym`
-    indices is the initial list of dummy indices
+    see `double_coset_can_rep` for the description of ``dummies`` and ``sym``
+    ``indices`` is the initial list of dummy indices
```

Comment:
```suggestion
    see ``double_coset_can_rep`` for the description of ``dummies`` and ``sym``.
    ``indices`` is the initial list of dummy indices.
```

Reply:
Also, add a period at the end of the previous line.

Reply:
Done.